molecule/default/roles/test_glauth/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Pre-install Debian package
+  ansible.builtin.apt:
+    # yamllint disable-line rule:line-length
+    deb: https://github.com/ASSEKURANSA/glauth/releases/download/debian%2F2.3.2-0_exp5/glauth-binary-dist_2.3.2-0.exp5_amd64.deb
+    state: present
+    update_cache: false
+  register: apt_glauth
+  changed_when: apt_glauth.changed
```

Comment:
This is redundant, that should be the default behaviour

Reply:
Oops, good catch. Removed (as well as `register`).